tests/full-disk-encryption.spec.ts

```diff
@@ -7,6 +7,10 @@ import { EncryptionPasswordPopup } from '../pages/encryption-password-popup';
 import { UsersPage } from '../pages/users-page';
 import { DefineUserPage } from '../pages/define-user-page';
 import { ConfigureRootPasswordPage } from '../pages/configure-root-password-page';
+import { ConfirmInstallationPage } from '../pages/confirm-installation-page';
+import { InstallationProgressPage } from '../pages/installing-page';
+import { CongratulationsPage } from '../pages/installation-finisihed-page';
```

Comment:
please use the same naming for the classes and the files, otherwise is confusing.

Reply:
s/finisihed/finished/g